docs/serializer-formatter.rst

```diff
@@ -128,40 +129,6 @@ The Avro schema for the value will be registered under the subject *recordName-v
 *recordName* is the name of the Avro record. Because of this, the same Avro record type shouldn't
 be used in more than one topic.
 
-In the following example, we send a message with key of type ``string`` and value of type Avro record
```

Comment:
Maybe not relevant to this particular patch, but it looks like we don't have a code example with the new consumer; just the old high level consumer